File: plugins/devkit/devkit-java-tests/testData/inspections/useJBColorFix/UseJBColorConstantStaticImportInVariable.java
import java.awt.Color;
import static java.awt.Color.BLACK;

class UseJBColorConstantStaticImportInVariable {
  void any() {
    Color myColor = <warning descr="'java.awt.Color' used instead of 'JBColor'">BL<caret>ACK</warning>;
  }
}

File: plugins/devkit/devkit-kotlin-tests/testData/inspections/useJBColorFix/UseJBColorConstantStaticImportInVariable.kt
import java.awt.Color.BLACK

class UseJBColorConstantStaticImportInVariable {
  fun any() {
    @Suppress("UNUSED_VARIABLE")
    val myColor = <warning descr="'java.awt.Color' used instead of 'JBColor'">BL<caret>ACK</warning>
  }
}
